src/callbacks_stage/positivity_zhang_shu_dg3d.jl

```diff
@@ -5,6 +5,22 @@
 @muladd begin
 #! format: noindent
 
+@inline function compute_u_mean(u::AbstractArray{<:Any, 5}, mesh::AbstractMesh{3},
+                                equations, dg::DGSEM, weights, inverse_jacobian,
+                                element)
+    u_mean = zero(get_node_vars(u, equations, dg, 1, 1, 1, element))
+    total_volume = zero(eltype(u))
+    for k in eachnode(dg), j in eachnode(dg), i in eachnode(dg)
+        volume_jacobian = abs(inv(get_inverse_jacobian(inverse_jacobian, mesh,
+                                                       i, j, k, element)))
+        u_node = get_node_vars(u, equations, dg, i, j, k, element)
+        u_mean += u_node *
+                  weights[i] * weights[j] * weights[k] * volume_jacobian
```

Comment:
```suggestion
        u_mean += u_node * weights[i] * weights[j] * weights[k] * volume_jacobian
```

Reply:
I changed the implementation to avoid the repeated computation of the "node_volume"